package/MDAnalysis/lib/src/encore_cutils/cutils.pyx

```diff
@@ -0,0 +1,68 @@
+# cutils.pyx --- C-compiled Utility functions for encore package
+# Copyright (C) 2014 Wouter Boomsma, Matteo Tiberti
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#cython embedsignature=True
+
+"""
+Mixed Cython utils for ENCORE
+
+:Author: Matteo Tiberti, Wouter Boomsma, Tone Bengtsen
+:Year: 2015--2016
+:Copyright: GNU Public License v3
+:Mantainer: Matteo Tiberti <matteo.tiberti@gmail.com>, mtiberti on github                       """
+
+import numpy as np
+cimport numpy as np
+import cython
+
+cdef extern from "math.h":
+    double sqrt(double x)
+    
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+
+def PureRMSD(np.ndarray[np.float64_t,ndim=2] coordsi,
```

Comment:
If you are that concerned about speed and memory consumption why not use single precision floats. This is the standard that MDAnalysis uses and for MD simulations enough for most cases.